tests/utils.py

```diff
@@ -0,0 +1,77 @@
+import syft as sy
+import torch
+import syfertext
+from syfertext.tokenizer import Tokenizer
+from syfertext.vocab import Vocab
+import pickle
+from syft.generic.string import String
+from syfertext.utils import hash_string
+import random
+import numpy as np
+
+
+def get_test_language_model():
+    nlp = syfertext.create(model_name="syfertext_sentiment")
+    # Create the tokenizer
+    tokenizer = Tokenizer()
+
+    vocabulary = [
```

Comment:
The list of words should also include all the words that are used in the tests. I suggest actually that you take the list of hashes from 'syfertext_en_lang_web_lg'. 

You can create a file containing this list of hashes, you put it in tests/vocab_hashes.py. This way we are sure we have all the necessary words.

Testing using this locally created language model will be changed again once @Nilanshrajput integrated the grid nodes.

Reply:
ok I will make the changes